exercises/queen-attack/.meta/template.j2

```diff
@@ -0,0 +1,38 @@
+{%- import "generator_macros.j2" as macros with context -%}
+{{ macros.header(imports=['Queen']) }}
+
+class {{ exercise | camel_case }}Test(unittest.TestCase):
+{% set suite_tests = cases -%}
+
+{% for cases in suite_tests -%}
+    # {{ cases["description"] }}
+    {% for case in cases["cases"] -%}
+    def test_{{ case["description"] | to_snake }}(self):
+        {% if case["property"] == "canAttack" -%}
+        {% set white_queen = case["input"]["white_queen"]["position"] -%}
+        {% set black_queen = case["input"]["black_queen"]["position"] -%}
+        self.assertIs(Queen({{ white_queen["row"] }}, {{ white_queen["column"] }}).can_attack(Queen({{ black_queen["row"] }}, {{ black_queen["column"] }})), {{ case["expected"] }})
+        {% else -%}
+        {% set queen = case["input"]["queen"]["position"] -%}
+        {% if case["expected"] == 0 -%}
+        try:
+            Queen({{ queen["row"] }}, {{ queen["column"] }})
+        except ValueError:
+            self.fail("Unexpected Exception")
```

Comment:
`unittest`/`pytest` will fail the test if an unexpected error is thrown; no need to do this manually.

Reply:
I recognize that the original did this; it shouldn't have. It should be enough to just call `Queen` without the `try`/`except`.

Reply:
Should I keep this test case? I know that this case checks if Queen has a valid position, but how could this be checked?

Reply:
The functionality is checked implicitly: if no exception is throw by `Queen(2, 2)`, the test case is a pass.

The test may be a little redundant, as later tests would fail if `Queen(validRow, validColumn)` throws an exception, but the test is canonical, so I'd prefer to keep it.